disnake/http.py

```diff
@@ -2125,10 +2125,18 @@ def edit_guild_welcome_screen(
     # Application commands (global)
 
     def get_global_commands(
-        self, application_id: Snowflake
+        self,
+        application_id: Snowflake,
+        *,
+        with_localizations: bool = True,
     ) -> Response[List[interactions.ApplicationCommand]]:
+        params: Dict[str, Any] = {}
+        if with_localizations:
```

Comment:
could you leave a comment why this is like this? (the api currently doesn't care about the value of with_localizations, just if it exists or not

Reply:
d30d667debd5840864810ebf5281680401f7f82f